### PR TITLE
Plugin for world map

### PR DESCRIPTION
**Pull request by kevalbhatt**

### CODE REVIEW

File: files/jquery.world-map/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "jquery.world-map",
+  "repo": "kevalbhatt/WorldMapGenerator",
+  "files": {
+    "include": ["WorldMapGenerator.min.js", "WorldMapGenerator.js", "example.html"]
```
Comment (megawac) [inline files/jquery.world-map/update.json]: why example.html?

**Comment by kevalbhatt** [comment]
i just removed example.html

**Comment by jimaek** [comment]
Can you please rename the directory to `jquery.worldmapgenerator` to match your plugin's name?

**Comment by kevalbhatt** [comment]
i changed the name to jquery.worldmapgenerator

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**